verifier/src/report_body.rs

```diff
@@ -82,6 +83,15 @@ impl<T> EqualityVerifier<T> {
     }
 }
 
+impl<T> Display for EqualityVerifier<T>
+where
+    T: SpacedStructName + Display,
+{
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        write!(f, "The {} is {}", T::spaced_struct_name(), self.expected)
```

Comment:
:thought_balloon: : If I'm understanding correctly we're still going to want a tree like output for what was verified.
It seems like this saying "The X is 3" is a verification output and not a verifier display.
IFF the plan is to use a tree output then the output in failure conditions would look like
```console
- [ ] The X is 3
  - [ ] The X was 2
```
Perhaps this should be something like "The expected {} is {}" or "The {} should be {}" so that the failed output would be
```
- [ ] The X should be 3
  - [ ] The actual X was 2
```
Then a success version being
```
- [x] The X should be 3
  - [x] The actual X was 3
```

Reply:
Agreed, will change according to the comment below.